### DISCUSSION TITLE
Upgrade Phoniebox Classic to Phoniebox +Spotify

### DISCUSSION THREAD

**Discussion by jonach66** [MiczFlor/RPi-Jukebox-RFID#1283] [Q&A]
Hi there!

I had some issues with audio output after installation via script for the +SPOTIFY Version, so I installed CLASSIC first to do some testing, and then upgraded to the +SPOTIFY version manually -> now audio is working properly.

BUT the Phoniebox web interface is not showing the **Spotify-Tab** (but Mopidy-Iris is running and playing ;-) ), I still see the CLASSIC version of Phoniebox web interface.

How to "upgrade" the Phoniebox web interface from CLASSIC to +SPOTIFY?

Thank you for any hints in advance!
Jo

**Comment by MiczFlor** [comment]
This is a good catch...
In this line, you can see how the Web App decides what to print in the header:
https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/0f7e9eb23c93c876c3343750d527fa137768efe0/htdocs/inc.header.php#L168
In this line you see where it get's the info from:
https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/0f7e9eb23c93c876c3343750d527fa137768efe0/htdocs/inc.header.php#L142
Which means: if you open the file `settings/global.conf` you can write whatever you want for `edition` to appear in the header.
If you make that change manually, you should also change the content in `/settings/edition` because whenever the Phoniebox creates the `global.conf` file, it will look there for what it's meant to print.
